dfm/description/description_pattern_lists.py

```diff
@@ -0,0 +1,89 @@
+occupation_pattern_list = [
+    "revisor",
+    "administrator",
+    # "rådgiver",  # Matches many non-occupation tokens, maybe use lemmatizer?
```

Comment:
Lemmatizer would be too slow. Not an option.
```suggestion
    # "rådgiver",  # Matches many non-occupation tokens
```